src/CentralApi/Program.cs

```diff
@@ -23,6 +23,7 @@
 if (builder.Environment.IsDevelopment() == false)
     builder.Services.AddHostedService<IndexCreationService>();
 
+builder.RegisterInflux();

```

Comment:
@appmdev, you are adding the reference to influxdb in CentralApi, but not using it, and you didn't add a correct configuration in `appsettings.json` in CentralApi. Maybe you could remove it?

Reply:
removed